src/stringify.js

```diff
@@ -160,7 +169,13 @@ module.exports = function (options) {
     return through(write, end);
   }
 
-  return browserifyTransform;
+  if (typeof file !== 'string') {
+    // Factory: return function (first argument is options)
+    options = file;
```

Comment:
Is this assignment necessary? It appears to do nothing with options after this...